lnrpc/routerrpc/router_server.go

```diff
@@ -396,3 +407,81 @@ func marshallChannelUpdate(update *lnwire.ChannelUpdate) *ChannelUpdate {
 		FeeRate:         update.FeeRate,
 	}
 }
+func (s *Server) LookupPayment(request *LookupPaymentRequest,
```

Comment:
Missing `godoc` comment as well as some breathing room between the function above.